### PR TITLE
Various corrections in README.md

### PR DESCRIPTION
**Pull request by RomainSanchez**
Nice bundle

---
This change is [Reviewable](https://reviewable.io/reviews/redco/sonata-admin-tree-bundle/3)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Install it via composer
 composer install redcode/tree-bundle
```
Comment (theholocoder) [inline README.md]: You can change this to `composer require redcode/tree-bundle`

**Review by theholocoder — COMMENTED** [review]
Line 26 you can also change `composer install` to `composer require`

File: README.md (hunk 1 above)
Comment (maZahaca) [reply README.md]: Nice suggestion!
@RomainSanchez please have a look and I will merge these changes

**Comment by RomainSanchez** [comment]
Done

**Comment by maZahaca** [comment]
@RomainSanchez Great! Thanks!

**Comment by RomainSanchez** [comment]
You're welcome.If you guys are heavy sonata admin users i invite you to check out https://github.com/blast-project/CoreBundle.I contributed to the development of this bundle and the other bundles of the blast project and would love to get feedback on it.(I originally made this pull request because i was considering integrating your bundle into BlastBaseEntitiesBundle, maybe you could help me with that)

**Comment by theholocoder** [comment]
Maybe we could add to the README file to do a `app/console assets:install --symlink` after adding the bundle to `AppKernel` ? I just got tricked, the tree view wasn't loading at all.

I'll checkout blast-project/CoreBundle later.

**Comment by maZahaca** [comment]
I would be appreciate if you decide to use this "kind of software" in your projects. It was done in one night just because of argent need of it.
I've spent few years on Sonata Admin and have an active project with it.
If you have any questions you can contact me.

**Pull request merged**